preflight.go

```diff
@@ -1,12 +1,26 @@
 package libyaml
 
 type CustomRequirement struct {
-	ID      string         `yaml:"id" json:"id" validate:"required,customrequirementidunique"`
-	Message Message        `yaml:"message" json:"message"`
-	Details *Message       `yaml:"details,omitempty" json:"details,omitempty"`
-	When    BoolString     `yaml:"when,omitempty" json:"when,omitempty"`
-	Results []CustomResult `yaml:"results" json:"results" validate:"required,min=1,dive"`
-	Command CustomCommand  `yaml:"command" json:"command"`
+	ID         string             `yaml:"id" json:"id" validate:"required,customrequirementidunique"`
+	Message    Message            `yaml:"message" json:"message"`
+	Details    *Message           `yaml:"details,omitempty" json:"details,omitempty"`
+	When       BoolString         `yaml:"when,omitempty" json:"when,omitempty"`
+	Results    []CustomResult     `yaml:"results" json:"results" validate:"required,min=1,dive"`
+	Command    CustomCommand      `yaml:"command" json:"command"`
+	Kubernetes *KubernetesCommand `yaml: "kubernetes,omitempty" json:"kubernetes,omitempty"`
```

Comment:
This diverges with how native scheduler handles scheduling commands using the `CustomCommand.Data` interface{} type. Im not saying that its the right or wrong way but i would prefer we be consistent.

https://github.com/replicatedcom/replicated/blob/master/scheduler/native/preflight/commands/raw.go#L19-L31